test/js/studyResultsContainer.test.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import StudyResultsContainer, { StudiesResults } from '../../app/javascript/components/StudyResultsContainer'
+import { shallow } from 'enzyme'
+import Tab from 'react-bootstrap/lib/Tab'
+
+describe('<StudyResultsContainer/> rendering>', () => {
+  it('should render <StudyResultsContainer/> elements', () => {
+    const studyResultsContainer = shallow(<StudyResultsContainer/>)
```

Comment:
same comment as above -- if you want to keep these expects, add a comment that they're snapshot-like